test.sh

```diff
@@ -43,15 +43,11 @@ if [ ! -z "$TRAVIS" ] || [ ! -z "$GH_ACTIONS" ]; then
     cat context/instance/app.conf
   )
 fi
-set -m; ./dev-start.sh & set +m  # Without job control, I had trouble killing parent and children.
-PID=$!
-server_up 5001  # Not really needed: Cypress will wait for response.
-end-to-end/test.sh
-kill -TERM -$PID  # Kill dev server processes
-end dev-start
+end copy-app-conf
 
 
 start flake8
+pip install -r context/requirements.txt > /dev/null
 # Unit tests require dev dependencies beyond requirements.txt.
 pip install -r context/requirements-dev.txt > /dev/null
```

Comment:
If we're getting rid of the weird `dev-start.sh` on CI, installs should probably just move outside this script.

Reply:
I can move the installs to the yml files for Github Actions and Travis. It would also then require you to run the installs before running this script locally. Should I wait until we decide between the two options?

Reply:
From stand-up: we'll move the installs out of test.sh. Those running `test.sh` locally can be entrusted to make installs.